src/components/Welcome.jsx

```diff
@@ -6,8 +6,7 @@ class Welcome extends Component {
         return (
         <div>
             <h1>You are welcome at Barefoot nomad</h1>
-			<Link to='/login'><h1>Click me to login</h1></Link>
-            <Link to='/redux'><h1>Click check redux</h1></Link>
+			<Link to='ForgotPassword'><h4>forgot password</h4></Link>
```

Comment:
i don't think we need this line. the link that someone will use to go to forgot password will come from login page. even the whole file is not needed.

Reply:
@Jajabenit250 will come from the login page that is for just checking how it works locally.

Reply:
@mujohn26 I think you can remove it. because the code is going to be merged. and check how it works locally by typing localhost/forgotpassword ...

Reply:
@Jajabenit250  Yeah I have removed it